holidays/countries/equatorial_guinea.py

```diff
@@ -0,0 +1,115 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import JAN
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON, SAT_TO_PREV_FRI
+
+
+class EquatorialGuinea(
+    InternationalHolidays, ObservedHolidayBase, ChristianHolidays, StaticHolidays
+):
+    """Equatorial Guinea holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Equatorial_Guinea>
+        * <https://www.timeanddate.com/holidays/guineaecuatorial/>
+        * <https://gq.usembassy.gov/holiday-calendar/>
+        * [President's Day](https://www.officeholidays.com/holidays/equatorial-guinea/equatorial-guinea-presidents-day)
+        * [AFCON Victory Holiday](https://www.timeanddate.com/holidays/equatorialguinea/afcon-victory-vs-ivory-coast)
```

Comment:
To be in-line with the upcoming #2504 changes
```suggestion
        * <https://en.wikipedia.org/wiki/Public_holidays_in_Equatorial_Guinea>
        * <https://web.archive.org/web/20250503042253/https://www.timeanddate.com/holidays/guineaecuatorial/>
        * <https://web.archive.org/web/20250503042610/https://gq.usembassy.gov/holiday-calendar/>
        * [President's Day](https://web.archive.org/web/20250503042339/https://www.officeholidays.com/holidays/equatorial-guinea/equatorial-guinea-presidents-day)
        * [AFCON Victory Holiday](https://web.archive.org/web/20250503042642/https://www.timeanddate.com/holidays/equatorialguinea/afcon-victory-vs-ivory-coast)
```